samples/apps/copilot-chat-app/scripts/3_init-frontend.cmd

```diff
@@ -0,0 +1,31 @@
+@echo off
+
+cd ..\WebApp
+
+if "%1" == "" (
+    goto :runapp
+)
+
+:: Create the .env file
+del .\.env
+echo REACT_APP_BACKEND_URI=https://localhost:40443/ >> .\.env
+
+if "%2" == "msa" (
+    echo REACT_APP_AAD_AUTHORITY=https://login.microsoftonline.com/9188040d-6c67-4c5b-b112-36a304b66dad >> .\.env
+) else (
+    if "%2" == "msft" (
```

Comment:
Do we want to include something that is only specific to internal Microsoft users?